src/index.html

```diff
@@ -2,11 +2,363 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a class="header__img" href="#">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__items list">
+          <li class="list__item item">
+            <a
+              href="#"
+              class="item__link
+              is-active"
+            >Apple</a>
```

Comment:
Check the rules of [HTML FORMATTING](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting) and correct all code according to these rules
```suggestion
            <a
              href="#"
              class="item__link
              is-active"
            >
              Apple
            </a>
```